dataset-metadata/metadata.ts

```diff
@@ -1,4 +1,24 @@
-import { datasetLabels } from '../browser/src/datasets.ts'
```

Comment:
Good call changing this to be more clearly named.

Reply:
part of my overall scheme for the branch that this was spun off from, is to dismantle `browser/src/datasets.ts` and move the usable parts to this new `dataset-metadata` project.